src/App.tsx

```diff
@@ -15,35 +20,91 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+function getGoods(
+  goods: string[],
+  sortField: SortType | '',
```

Comment:
The `sortField` parameter is expected to be of type `SortType | ''`, but the `sortValue` state is initialized to `-1`. This inconsistency can cause runtime errors. Consider initializing `sortValue` to `''` or a valid `SortType` value.